src/redux/reducers/todoReducer.js

```diff
@@ -25,13 +29,30 @@ export const todoReducer = (state = initialState, action) => {
                 isLoading: false,
                 todos: action.payload
             }
-        
+
         case LOAD_TODOS_FAILURE:
             return {
                 ...state,
                 error: action.payload.message
             }
 
+        case ADD_TODO_START:
```

Comment:
would you please take a **FOCUS** look on your reducers, you have done them correct before ;)